docs/database/connecting.md

```diff
@@ -0,0 +1,37 @@
+# Connect to MySQL inside of a Docker container
+
+There may come a time when developing or managing a Monica instance that you
+would like to connect directly to the MySQL database being used.
+
+For example you may want to
+- alter test data
+- generate custom metrics
+- ...
+
+---
+
+<!-- TODO(review): Is there a better way to express/show these file changes? -->
+1. Stop your Monica and MySQL containers if they are running
+1. Bind port 3306 in your container to 3306 on your host machine
+    <!-- TODO(review): Are these changes safe to include directly in-tree? -->
```

Comment:
My understanding is that this should be safe so long as the host machine itself doesn't expose 3306 (unless you _want_ other machines to be able to connect to the db).